packages/github/package.json

```diff
@@ -27,13 +27,15 @@
   "scripts": {
     "test": "jest",
     "build": "tsc",
+    "format": "prettier --write **/*.ts",
+    "format-check": "prettier --check **/*.ts",
     "tsc": "tsc"
   },
   "bugs": {
     "url": "https://github.com/actions/toolkit/issues"
   },
   "dependencies": {
-    "@octokit/graphql": "^2.0.1",
+    "@octokit/graphql": "^4.3.1",
     "@octokit/rest": "^16.15.0"
```

Comment:
Consider bumping this, we are about a year behind and there are a number of bugfixes and feature updates since then.